src/EchoIt/JsonApi/Handler.php

```diff
@@ -404,33 +405,28 @@ protected function handleSortRequest($cols, $model)
      * @return array
      * @throws Exception
      */
-    protected function parseRequestContent($content, $type)
-    {
-        $content = json_decode($content, true);
-        if (empty($content['data'])) {
+    protected function parseRequestContent( $content, $type ) {
+        $content = json_decode( $content, true );
+        $data    = $content['data'];
```

Comment:
That's not good. You need to check if `data` is there in the first place; the original code did this properly.